spec.html

```diff
@@ -27244,8 +27244,7 @@ <h1>Evaluate ( ): a Promise</h1>
                 1. Assert: _module_.[[Status]] is either ~evaluating-async~ or ~evaluated~.
                 1. Assert: _module_.[[EvaluationError]] is ~empty~.
                 1. If _module_.[[Status]] is ~evaluated~, then
-                  1. NOTE: This implies that evaluation of _module_ completed synchronously.
-                  1. Assert: _module_.[[AsyncEvaluationOrder]] is ~unset~.
+                  1. Assert: _module_.[[AsyncEvaluationOrder]] is ~unset~ or ~done~.
```

Comment:
Can you re-add the note for posterity? Not with the original wording, which was wrong, but maybe something like
```suggestion
                  1. Assert: _module_.[[AsyncEvaluationOrder]] is ~unset~ or ~done~.
                  1. NOTE: _module_.[[AsyncEvaluationOrder]] is ~done~ if and only if _module_ had already been evaluated and that evaluation was asynchronous.
```